polytope/polytope.py

```diff
@@ -1307,14 +1308,11 @@ def volume(polyreg):
         N = 10000
 
     l_b, u_b = polyreg.bounding_box
-    x = np.tile(l_b, (1, N)) +\
-        np.random.rand(n, N) *\
-        np.tile(u_b-l_b, (1, N))
-    aux = np.dot(polyreg.A, x) -\
-        np.tile(
-            np.array([polyreg.b]).T,
-            (1, N)
-        )
+    x = (np.tile(l_b, (1, N))
+         + np.random.rand(n, N)
+         * np.tile(u_b-l_b, (1, N)))
+    aux = (np.dot(polyreg.A, x)
```

Comment:
this change is distinct from the proposals in [our thread of comments](https://github.com/tulip-control/polytope/pull/33#pullrequestreview-16908107) on a prior commit. In any case, I am happy with this one.